app-network/shadowsocks-rust/autobuild/defines

```diff
@@ -6,12 +6,5 @@ USECLANG=1
 
 PKGDES="A Rust port of shadowsocks, lightweight yet secured SOCKS5 proxy"
 
-ABSPLITDBG=0
-# Since ring does not support ppc64el and loongson3 architectures, use local-http-native-tls to avoid this dependency 
-CARGO_AFTER="--features local-http-native-tls,local-tunnel,local-redir,local-dns,trust-dns,stream-cipher,aead-cipher-extra"
```

Comment:
In previous versions, lots of these features weren't enabled by default, why removing all cargo parameters in this version?

Reply:
Now shadowsocks-rust provides its own default feature set, which includes more than those we manually specified.